lib/Doctrine/Export/Oracle.php

```diff
@@ -580,7 +580,7 @@ public function getIndexDeclaration($name, array $definition)
                 $type = strtoupper($definition['type']);
             } else {
                 throw new Doctrine_Export_Exception(
-                    'Unknown type ' . $definition['type'] . ' for index ' . $name
+                    'Unknown type ' . $definition['type'] . ' for index ' . $name
```

Comment:
This was a weird non-printable character that the PHP70Migration in php-cs-fixer turned into a unicode escape sequence. I just removed the character and put a space.